extensions/global-media-key/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "global-media-key",
+  "title": "Media Key Emulate",
+  "description": "Emulate keyboard media keys press. Reassign hotkey for media keys.",
+  "icon": "extension_icon.png",
+  "author": "douo",
+  "categories": [
+    "System"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "play",
+      "description": "Toggle Play/Pause",
+      "title": "Play/Pause",
+      "icon": "play.png",
+      "mode": "no-view"
+    },
+    {
+      "name": "next",
+      "description": "Toggle Play/Pause",
+      "title": "Next",
+      "icon": "next.png",
+      "mode": "no-view"
+    },
+    {
+      "name": "previous",
+      "description": "Toggle Play/Pause",
+      "title": "Previous",
+      "icon": "previous.png",
+      "mode": "no-view"
+    },
+    {
+      "name": "fast",
+      "description": "Toggle Play/Pause",
+      "title": "Fast",
+      "icon": "fast.png",
+      "mode": "no-view"
+    },
+    {
+      "name": "rewind",
+      "description": "Toggle Play/Pause",
+      "title": "Rewind",
+      "icon": "rewind.png",
+      "mode": "no-view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.48.9"
```

Comment:
Do you mind updating to the latest API version?

Reply:
Sure!